README.md

```diff
@@ -1,44 +1,47 @@
-This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+# Moltin &mdash; React Demo Store
 
-You can find a guide to some of the most common tasks [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
- 
----
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app). You can find a guide to some of the most common tasks [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
 
-## Deploying the app with Heroku
+## Development
 
-[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
+```bash
+git clone https://github.com/moltin/react-demo-store.git
+cd react-demo-store
+yarn #or npm install
+yarn start # or npm start
+```
+
+Note: You will want to change the `client_id` inside `src/moltin.js` with your own moltin store credentials.
+
+This demo store uses the Redux "[ducks](https://github.com/erikras/ducks-modular-redux)" approach to bundling reducers and actions.
+
+## Deployment
+
+### Heroku
 
----
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
 
-## Deploying the app with Docker
+### Docker
 
 1. [Download and install docker](https://docs.docker.com/engine/installation/)
 2. Make sure docker is running locally
-3. Run `docker build -t lamp ` at command line
-4. Run the docker image with the command `docker run -p 
-5000 IMAGE_ID` where `IMAGE_ID` is the image ID shown in the result of step 3.
+3. Run `docker build -t lamp` at command line
+4. Run the docker image with the command `docker run -p 5000 IMAGE_ID` where `IMAGE_ID` is the image ID shown in the result of step 3.
 5. Access your app on port 5000
 
----
+### Digital Ocean
 
-## Deploying the app with Digital Ocean
 **Not ready**
-[![Install on DigitalOcean](http://installer.71m.us/button.svg)](http://installer.71m.us/install?url=https://github.com/matthew1809/ILoveLamp-React)
-
----
 
-## Running this app locally
-1. Clone this repository & `cd` into it.
-2. Run `npm install`.
-3. In `src/utils/moltin.js`, on line 3, replace `client_id` with your own moltin client_id
-4. Run `npm start`.
+[![Install on DigitalOcean](http://installer.71m.us/button.svg)](http://installer.71m.us/install?url=https://github.com/matthew1809/ILoveLamp-React)
```

Comment:
Does this link to an up to date version of this codebase?

Reply:
This link is tagged as **not ready** in the above message so it might be worth removing it until I get to the deploy stage and put it back "working".